### ISSUE TITLE
Confusing text in Step 5 of tutorial

### ISSUE THREAD

**Issue by robguinness** [apollographql/apollo#696]
In [step 5](https://www.apollographql.com/docs/tutorial/client/) of the tutorial, it states:

> If you didn't complete the server portion, you can use the uri from the code below. Otherwise, use your own deployment's URL, which may be different than the one below. 

I found this text confusing. If one does complete the server portion, the uri is actually the same as in the referenced code, and it's not explicit in the server source code what the uri is...so for a newbie this is a bit confusing.

**Comment by LongLiveCHIEF** [comment]
Yeah, the whole marketing plug which is step 4 totally throws the flow of the tutorial. 

I wish they'd call it `Step 3a` or something like that and say something like "if you'd like to see how Graph Manager and other Apollo cloud services can benefit you, go to Step 3A, otherwise proceed to Step 4: Connect your API to a Client"
